RubberduckTests/Refactoring/ImplicitTypeToExplicit/ImplicitTypeToExplicitRefactoringActionConstantTests.cs

```diff
@@ -157,6 +157,24 @@ public void ConstantTypedByFunctionType(string procedureType, string getToken)
 Public {procedureType} {getToken}AValue() As Double
     AValue = MY_CONSTANT
 End {procedureType}
+";
+            var refactoredCode = RefactoredCode(inputCode, NameAndDeclarationTypeTuple(targetName));
+            StringAssert.Contains($"{targetName} As {expectedType}", refactoredCode);
+        }
+
+        [TestCase("5 & 5", "String")]
+        [TestCase("Null & Null", "Variant")]
```

Comment:
Wouldn't this one be coerced into an empty string?

Reply:
The result of assigning `Null & Null` to a `String` variable results in an 'Invalid use of null' error.  If assigned to a `Variant` variable, the result is `Variant/Null`.  (MS-VBAL 5.6.9.4 page 148)

Reply:
How does this apply when using `vbNullString`? Does that resolve to an Empty string?

Reply:
Yes...The result of `vbNullString` & `vbNullString` is an empty String.